shell/browser/electron_browser_context.h

```diff
@@ -221,9 +221,9 @@ class ElectronBrowserContext : public content::BrowserContext {
   // Initialize pref registry.
   void InitPrefs();
 
-  bool DoesDeviceMatch(const base::Value& device,
-                       const base::Value* device_to_compare,
-                       blink::PermissionType permission_type);
+  static bool DoesDeviceMatch(const base::Value& device,
+                              const base::Value* device_to_compare,
+                              blink::PermissionType permission_type);
```

Comment:
If this doesn't need to actually be a member function, can we pull it out into a static function in the .cc only?

Reply:
Yes, good idea. Moved in b1ef1feef45428da5d12d5f5471782d4cc2dbfbb.